src/content/recipes/vuetify.md

```diff
@@ -0,0 +1,196 @@
+
+setup((app) => {
+  // Registers your app's plugins into Storybook
+  registerPlugins(app);
+});
+```
+
+Here `registerPlugins` loads Vuetify’s fonts and registers all of its components with Storybook’s Vue app.
+
+## Using Vuetify Components
+
+Let’s update some of our example components to use Vuetify instead. Open up the Button component in `./src/stories/button.vue`.
+
+![Unchanged example button component from Storybook init](https://user-images.githubusercontent.com/18172605/207120859-0383d01a-6448-4327-94c9-dcb1ec86f868.png)
+
+Currently, it’s not looking very Vuetiful so let’s make some changes. Replace the contents of `./src/stories/Button.vue` with the following code:
+
+```vue
+<template>
+  <v-btn type="button" color="primary" @click="onClick" :variant="variant" :size="size">{{
+    label
+  }}</v-btn>
+</template>
+
+<script>
+import { reactive, computed } from 'vue';
+
+export default {
+  name: 'my-button',
+
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    primary: {
+      type: Boolean,
+      default: false,
+    },
+    size: {
+      type: String,
+      validator: function (value) {
+        return ['small', 'large'].indexOf(value) !== -1;
+      },
+    },
+    backgroundColor: {
+      type: String,
+    },
+  },
+
+  emits: ['click'],
+
+  setup(props, { emit }) {
+    props = reactive(props);
+    return {
+      onClick() {
+        emit('click');
+      },
+      variant: computed(() => (props.primary ? 'flat' : 'outlined')),
+    };
+  },
+};
+</script>
+```
+
+Now looking back at Storybook, the Vuetify button is being used. It even changed in the page-level stories.
+
+![Converting the example button into a Vuetify button](https://user-images.githubusercontent.com/18172605/207120996-cdd40459-97f7-4e40-9782-719c45c38d11.gif)
+
+## Add a theme switcher tool using globalTypes
+
+Vuetify comes out of the box with a light and dark theme that you can override or add to. To get the most out of your stories, you should have a way to toggle between all of your themes.
+
+![Switching to Vuetify's dark theme in Storybook](https://user-images.githubusercontent.com/18172605/207121142-dbc27018-02d1-438d-b3d1-1d45e265e16a.gif)
+
+To add our switcher, declare a global variable named `theme` in `.storybook/preview.js` and give it a list of supported themes to choose from.
+
+```js
+// .storybook/preview.js
+
+export const globalTypes = {
+  theme: {
+    name: 'Theme',
+    description: 'Global theme for components',
+    toolbar: {
+      icon: 'paintbrush',
+      // Array of plain string values or MenuItem shape (see below)
+      items: [
+        { value: 'light', title: 'Light', left: '🌞' },
+        { value: 'dark', title: 'Dark', left: '🌛' },
+      ],
+      // Change title based on selected value
+      dynamicTitle: true,
+    },
+  },
+};
+```
+
+This code will create a new toolbar dropdown to select your desired theme for your stories.
```

Comment:
It looks like we're not super consistent in what we call these, but "toolbar menu" is [used more than once here](https://storybook.js.org/docs/react/essentials/toolbars-and-globals), and avoids the vague "dropdown" term.

```suggestion
This code will create a new toolbar menu to select your desired theme for your stories.
```